src/test/java/picard/sam/AbstractAlignmentMergerTest.java

```diff
@@ -199,6 +200,136 @@
+                //     <ssssssssSSSSMMMMMMMMMMMMMMM
+                //                  MMMMMMMMMMMMMMMSSSsssssssss>
+
+
+                {27, 14, 18, "8S19M", "15M1D3M9S", true, false, 18, 18, "12S15M", "15M12S", false,
+                        default27LongR1Bases, default27LongR2Bases, default27LongR1Bases, default27LongR2Bases, null, null,
+                        default27LongR1Qualities, default27LongR2Qualities,default27LongR1Qualities, default27LongR2Qualities , null,null },
+
+
+                // 123456789.123456789.123456789.123456789.123456789.123456789.123456789.123456789
+                //     <ssssssssMMMMMMMMMMMMMMMMMMM
+                //                  MMMMMMMMMMMMMMM---MMMsssssssss>
+                //          should become
+                // 123456789.123456789.123456789.123456789.123456789.123456789.123456789.123456789
+                //     <ssssssssSSSSMMMMMMMMMMMMMMM
+                //                  MMMMMMMMMMMMMMMSSSsssssssss>
+
+
+                {27, 14, 18, "8S19M", "15M3D3M9S", true, false, 18, 18, "12S15M", "15M12S", false,
+                        default27LongR1Bases, default27LongR2Bases, default27LongR1Bases, default27LongR2Bases, null, null,
+                        default27LongR1Qualities, default27LongR2Qualities,default27LongR1Qualities, default27LongR2Qualities , null,null },
+
+
+
+                //                  123456789.123456--789.123456789.123456789.123456789.123456789.123456789.123456789
+                //                                     MMMMMMMMMMMMMMMMMMsssssssss>
+                //                          <sssssMMiiMMMMMMMMMMMMMMMMMM
+                //                                  /\
+                //   with an insertion of two bases here should become
+                //                    123456789.123456789.123456789.123456789.123456789.123456789.123456789.123456789
+                //                                     MMMMMMMMMMMMMMMMMssssssssss>
+                //                          <sssssSSSSSMMMMMMMMMMMMMMMMM
+
+
+                {27, 18, 15, "18M9S", "5S2M2I18M", false, true, 18, 18, "17M10S", "10S17M", false,
+                        default27LongR1Bases, default27LongR2Bases, default27LongR1Bases, default27LongR2Bases, null, null,
+                        default27LongR1Qualities, default27LongR2Qualities,default27LongR1Qualities, default27LongR2Qualities , null,null },
+
+
+                //       123456789.123456789.123456789.1234--56789.123456789.123456789.123456789.123456789
+                //            <sssssssMMMMMMMMMMMMMMMMMMMM
+                //                        MMMMMMMMMMMMMMMMMiiMsssssss>
+                //                                         /\
+                //         with an insertion of two bases here should become
+                //       123456789.123456789.123456789.123456789.123456789.123456789.123456789.123456789
+                //            <ssssssSSSSSMMMMMMMMMMMMMMMM
```

Comment:
//            <ssssssSSSSSMMMMMMMMMMMMMMMM
should be
                //            <sssssssSSSSMMMMMMMMMMMMMMMM